docker-compose.yml

```diff
@@ -18,6 +18,9 @@ services:
       - "npm start"
   mojaloop-testing-toolkit-ui:
     image: mojaloop/ml-testing-toolkit-ui:v10.6.0
+    environment:
+      - API_BASE_URL=http://localhost:5050
+      - AUTH_ENABLED=FALSE
```

Comment:
I added the environment already. Can you please check for any duplication there?